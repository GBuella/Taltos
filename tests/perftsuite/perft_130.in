setboard 1nbk2nr/rppP1pp1/7p/8/8/7P/1qP1PPP1/1N1QKBNR w K - 16 9
echo 1nbk2nr/rppP1pp1/7p/8/8/7P/1qP1PPP1/1N1QKBNR w K - 16 9
qperft 1
qperft 2
qperft 3
qperft 4
