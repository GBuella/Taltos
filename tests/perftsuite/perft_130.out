1nbk2nr/rppP1pp1/7p/8/8/7P/1qP1PPP1/1N1QKBNR w K - 16 9
25
808
21822
739714
